### PR TITLE
fs: wc command added

### PR DESCRIPTION
**Pull request by patriki01**
Simple PR adding wc command with arguments.

### CODE REVIEW

File: pytest_mh/utils/fs.py [hunk 1]
```diff
@@ -442,3 +442,32 @@ def __gen_chattrs(
...
+        :param word: Print the word counts, defaults to False
+        :type word: bool, optional
+        :param bytes: Print the byte counts, defaults to False
+        :type bytes: bool, optional
+        :return: Result of process
+        :rtype: SSHProcessResult
+        """
+        args = []
+        if lines:
+            args = ["-l"]
```
Comment (pbrezina) [inline pytest_mh/utils/fs.py:465]: args.append here as well

File: pytest_mh/utils/fs.py [hunk 2]
```diff
@@ -442,3 +442,32 @@ def __gen_chattrs(
             cmds.append(f"chgrp '{group}' '{path}'")
 
         return " && ".join(cmds)
+
+    def wc(self, file: str, lines: bool = False, word: bool = False, bytes: bool = False) -> SSHProcessResult:
```
Comment (pbrezina) [inline pytest_mh/utils/fs.py:446]: There is also -m chars.

**Review by pbrezina — CHANGES_REQUESTED** [review]

File: pytest_mh/utils/fs.py (hunk 2 above)
Comment (patriki01) [reply pytest_mh/utils/fs.py]: added

File: pytest_mh/utils/fs.py (hunk 1 above)
Comment (patriki01) [reply pytest_mh/utils/fs.py]: fixed

**Comment by pbrezina** [comment]
Ack.

**Pull request merged**